webapi/src/main/java/org/sakaiproject/webapi/controllers/SitesController.java

```diff
@@ -175,9 +173,7 @@
         }).collect(Collectors.toList());
 
         Map<String, List<Map<String, Object>>> data = new HashMap<>();
-        data.put("favorites", sitesList);
+        data.put("favourites", sitesList);
```

Comment:
🤮

Reply:
I also favour the phonetics of favour. Favour sounds different to favor. To me favor sounds like it should be pronounced in an italian or spanish way, like sabor. Which is funny, seeing as sabor means flavour.

Reply:
So, you be sick all you want. Enjoy the flavour of that favour.

Reply:
:)

Reply:
However, if you insist, I will revert it to favorites.